Detectors/TOF/workflow/src/TOFClusterizerSpec.cxx

```diff
@@ -146,6 +170,28 @@ class TOFDPLClustererTask
     }
 
     for (int i = 0; i < row.size(); i++) {
```

Comment:
this will complain about comparison of signed and unsigned, use `unsigned i = 0...`

Reply:
mvd to unsigned